### PR TITLE
Write HTTPLogs for Whatsapp template syncing

### PR DESCRIPTION
**Pull request by norkans7**
Addresses https://github.com/rapidpro/rapidpro/issues/1106

### CODE REVIEW

File: temba/request_logs/models.py [hunk 1]
```diff
@@ -21,16 +21,18 @@ class HTTPLog(models.Model):
     REQUEST_DELIM = ">!>!>! "
     RESPONSE_DELIM = "<!<!<! "
 
-    # classifier type choices
+    # log type choices
     INTENTS_SYNCED = "intents_synced"
     CLASSIFIER_CALLED = "classifier_called"
     AIRTIME_TRANSFERRED = "airtime_transferred"
+    CHANNEL_REQUESTS = "channel_requests"
```
Comment (nicpottier) [inline temba/request_logs/models.py]: this should probably be more specific.. more like `whatsapp_templates_synced`

File: temba/request_logs/models.py [hunk 2]
```diff
@@ -21,16 +21,18 @@ class HTTPLog(models.Model):
...
     CLASSIFIER_CALLED = "classifier_called"
     AIRTIME_TRANSFERRED = "airtime_transferred"
+    WHATSAPP_TEMPLATE_SYNCED = "whatsapp_templates_synced"
 
     # possible log type choices and descriptive names
     LOG_TYPE_CHOICES = (
         (INTENTS_SYNCED, _("Intents Synced")),
         (CLASSIFIER_CALLED, _("Classifier Called")),
         (AIRTIME_TRANSFERRED, _("Airtime Transferred")),
+        (WHATSAPP_TEMPLATE_SYNCED, _("Whatsapp template Synced")),
```
Comment (rowanseymour) [inline temba/request_logs/models.py]: little bit of inconsistency with pluralization here - here and above I'd make it "templates"

File: temba/request_logs/models.py [hunk 3]
```diff
@@ -21,16 +21,18 @@ class HTTPLog(models.Model):
...
     CLASSIFIER_CALLED = "classifier_called"
     AIRTIME_TRANSFERRED = "airtime_transferred"
+    WHATSAPP_TEMPLATES_SYNCED = "whatsapp_templates_synced"
 
     # possible log type choices and descriptive names
     LOG_TYPE_CHOICES = (
         (INTENTS_SYNCED, _("Intents Synced")),
         (CLASSIFIER_CALLED, _("Classifier Called")),
         (AIRTIME_TRANSFERRED, _("Airtime Transferred")),
+        (WHATSAPP_TEMPLATES_SYNCED, _("Whatsapp templates Synced")),
```
Comment (rowanseymour) [inline temba/request_logs/models.py]: capital T

**Comment by norkans7** [comment]
@nicpottier I just added that in https://github.com/nyaruka/rapidpro/pull/2621/commits/fb458d7da03e4f1a96ab65f3d181ecdeee830645 and it is on a different PR https://github.com/nyaruka/rapidpro/pull/2621

**Pull request merged**